draft-ietf-tls-tls13.md

```diff
@@ -2746,13 +2711,18 @@ from the resumption master secret:
                             "resumption context", "", Hash.Length)
 ~~~~
 
-The client MAY use this PSK for future handshakes by including
-the ticket value in the "pre_shared_key" extension in its ClientHello
-({{pre-shared-key-extension}}) and supplying a suitable PSK cipher
-suite. Servers may send multiple tickets on a single connection, for
-instance after post-handshake authentication. For handshakes that
-do not use a resumption_psk, the resumption_context is a string of
-Hash.Length zeroes.
+The client MAY use this PSK for future handshakes by including the
+ticket value in the "pre_shared_key" extension in its ClientHello
+({{pre-shared-key-extension}}). Servers may send multiple tickets on a
+single connection, for instance after post-handshake
+authentication. For handshakes that do not use a resumption_psk, the
+resumption_context is a string of [[TBD: This is not safe now
+that we allow additional server signatures with PSK: 
+OPEN ISSUE https://github.com/tlswg/tls13-spec/issues/558]]
+
+Any ticket MUST only be resumed with a cipher suite that is identical
```

Comment:
Just to double-check, we can get away with this now because the PSK state is not part of the cipher suite, and we don't have to worry about taking the original cipher suite and converting it to its PSK analogue?